posthog/templates/surveys/public_survey.html

```diff
@@ -0,0 +1,939 @@
+        if (window.surveyAppearance) {
+            applySurveyAppearance(window.surveyAppearance);
+        }
+
+        // Load PostHog from CDN
+        !function (t, e) { var o, n, p, r; e.__SV || (window.posthog = e, e._i = [], e.init = function (i, s, a) { function g(t, e) { var o = e.split("."); 2 == o.length && (t = t[o[0]], e = o[1]), t[e] = function () { t.push([e].concat(Array.prototype.slice.call(arguments, 0))) } } (p = t.createElement("script")).type = "text/javascript", p.crossOrigin = "anonymous", p.async = !0, p.src = s.api_host.replace(".i.posthog.com", "-assets.i.posthog.com") + "/static/array.js", (r = t.getElementsByTagName("script")[0]).parentNode.insertBefore(p, r); var u = e; for (void 0 !== a ? u = e[a] = [] : a = "posthog", u.people = u.people || [], u.toString = function (t) { var e = "posthog"; return "posthog" !== a && (e += "." + a), t || (e += " (stub)"), e }, u.people.toString = function () { return u.toString(1) + ".people (stub)" }, o = "init Ie Ts Ms Ee Es Rs capture Ge calculateEventProperties Os register register_once register_for_session unregister unregister_for_session js getFeatureFlag getFeatureFlagPayload isFeatureEnabled reloadFeatureFlags updateEarlyAccessFeatureEnrollment getEarlyAccessFeatures on onFeatureFlags onSurveysLoaded onSessionId getSurveys getActiveMatchingSurveys renderSurvey canRenderSurvey canRenderSurveyAsync identify setPersonProperties group resetGroups setPersonPropertiesForFlags resetPersonPropertiesForFlags setGroupPropertiesForFlags resetGroupPropertiesForFlags reset get_distinct_id getGroups get_session_id get_session_replay_url alias set_config startSessionRecording stopSessionRecording sessionRecordingStarted captureException loadToolbar get_property getSessionProperty Ds Fs createPersonProfile Ls Ps opt_in_capturing opt_out_capturing has_opted_in_capturing has_opted_out_capturing clear_opt_in_out_capturing Cs debug I As getPageViewId captureTraceFeedback captureTraceMetric".split(" "), n = 0; n < o.length; n++)g(u, o[n]); e._i.push([i, s, a]) }, e.__SV = 1) }(document, window.posthog || []);
+
+        // Initialize PostHog with project configuration
+        const config = {
+            api_host: window.projectConfig.api_host,
+            disable_surveys_automatic_display: true,
+            debug: true,
+            advanced_disable_toolbar_metrics: true,
+            persistence: 'localStorage',
+            capture_pageview: false, // Don't capture pageviews for survey pages
+            capture_pageleave: false
+        };
+
+        // Add ui_host if available (for reverse proxy setups)
+        if (window.projectConfig.ui_host) {
+            config.ui_host = window.projectConfig.ui_host;
+        }
+
+        posthog.init(window.projectConfig.token, config);
+        const distinctId = new URLSearchParams(window.location.search).get('distinct_id');
+        if (distinctId) {
+            posthog.identify(distinctId);
+        }
```

Comment:
Users will need to pay for this event, which might not be expected/intentional.
Can we actually just mimic the identify behaviour without capturing the identify event?
what i mean is, the survey events should contain the 'distinctId' but we dont necessary need to capture an identify event, an idea is to init the SDK with the `bootstrap` object and the given `distinctId`